### PR TITLE
AC Test Fix - LRAC-14839 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LRAC-14839

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by joshitagaki — APPROVED** [review]
Thanks Vini! LGTM!

**Review by nkkwon — APPROVED** [review]
LGTM!

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro [hunk 1]
```diff
@@ -687,7 +687,9 @@ definition {
 	}
 
 	@summary = "Default summary"
-	macro viewAutocompleteSuggestions(notPresent = null, autocompleteText = null) {
+	macro viewAutocompleteSuggestions(notPresent = null,autocompleteText = null) {
+		var key_autocompleteText = ${autocompleteText};
+
```
Comment (manuferreira) [inline portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro:692]: Hey Vini, isn't the autocompleteText variable required for this function to work? if yes, I believe it shouldn't be null.

**Review by manuferreira — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro (hunk 1 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro:692]: Oh yeah, you're right. I think this was put by SF and I didn't remember to check. Thanks, Manu 🕵🏽‍♀️

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSegments.macro]: Thank you!

**Review by manuferreira — APPROVED** [review]
LGTM! Thanks VIni :detective:

**Comment by manuferreira** [comment]
ci:forward

**Pull request closed without merging**